src/assets/google-analytics.js

```diff
@@ -0,0 +1,16 @@
+function initGoogleAnalytics(i, s, o, g, r, a, m) {
+  i['GoogleAnalyticsObject'] = r;
+  i[r] = i[r] || function () {
+      (i[r].q = i[r].q || []).push(arguments)
+    }, i[r].l = 1 * new Date();
+  a = s.createElement(o),
+    m = s.getElementsByTagName(o)[0];
+  a.async = 1;
+  a.src = g;
+  m.parentNode.insertBefore(a, m);
+}
+
+if (window.ENV && window.ENV.GA_KEY) {
+  initGoogleAnalytics(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
+  ga('create', window.ENV.GA_KEY, 'auto');
+}
```

Comment:
Does this need to get added to `index.pug`? Looks like we're including it there in Publish and Metrics https://github.com/PRX/publish.prx.org/blob/master/src/index.pug#L21

Reply:
Yep, totally. I was thinking the PRX node server iterated through assets and did this automatically for some reason.